lib/graphiti/resource/remote.rb

```diff
@@ -38,6 +38,7 @@ def before_resolve(scope, query)
       # Forward all headers
       def request_headers
         {}.tap do |headers|
+          # TODO: Maybe handle this in graphiti-rails
```

Comment:
Yeah. These and others are definitely quick fixes that would ideally live in graphiti-rails. That said, they don't bother me too much as long as there aren't too many and they remain small. Up to you.

Reply:
I left them alone for now since they're small and I don't think it's terrible if they're left behind. That said, I think the long term approach is probably to expose a flag on Graphiti that can get set in graphiti-rails.